<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfessionChooseForm</class>
 <widget class="QWidget" name="ProfessionChooseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid ; 
    border-radius: 3px; 
	border-color:rgb(29,191,151);
    background: white;  
    color:rgb(29,191,151);
}

QPushButton:hover {
	border: 1px solid ; 
    border-radius: 3px; 
	border-color:rgb(29,191,151);
    background: rgb(128,243,215);  
    color:rgb(29,191,151);
}

QWidget {
	background:#FFFFFF;
}</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>50</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>IBM技术中心</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>50</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>50</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>中国西部边疆安全与发展协同创新中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>党委学生工作部（处）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>党委武装部（军事教研室）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>分析测试中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_50">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>750</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>公共管理学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>化学学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>化学基础实验教学中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_28">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>华西临床医学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_35">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>制造科学与工程学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>体育学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>化学工程学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_31">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>华西基础医学与法医学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>华西口腔医学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>华西公共卫生学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_52">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>750</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>历史文化学院（旅游学院）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_48">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>700</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>吴玉章学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>华西药学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_41">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>商学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_49">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>700</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>匹兹堡学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>国际关系学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>图书馆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_47">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>700</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>外国语学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_42">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>650</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>建筑与环境学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>实验室及设备管理处</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>工程训练中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>200</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>工程设计中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>心理健康教育中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>招生就业处</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_36">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>数学学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>文化科技协同创新研发中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_46">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>700</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>文新学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_44">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>650</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>法学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_45">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>700</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>材料科学与工程学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>校团委</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>灾后重建与管理学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>空天科学与工程学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_39">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>600</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>水利水电学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_34">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>生物治疗国家重点实验室</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>生命科学学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_51">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>750</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>物理科学与技术学院（核科学与工程技术学院）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>电工电子中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>电子信息学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>电气信息学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_43">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>650</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>轻纺与食品学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>网络空间安全学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_40">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>650</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>经济学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>计算机学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>计算机基础教学实验中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_37">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>600</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>艺术学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_38">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>600</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>高分子科学与工程学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>马克思主义学院</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>50</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>50</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>50</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>130</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>清空输入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
